### PR TITLE
Remove spaces from project folder/file names

### LINKED ISSUE #42: Remove spaces from project folder/file names
### DoD:

- remove spaces from all project folder/file names.
   -  Sleep Tracker -> SleepTracker
   - Sleep TrackerTests -> SleepTrackerTests
   - Sleep_TrackerTests -> SleepTrackerTests
   - Sleep TrackerUITests -> SleepTrackerUITests
   - Sleep_TrackerUITests.swift -> SleepTrackerUITests.swift
   - Sleep_TrackerUITestsLaunchTests.swift -> SleepTrackerUITestsLaunchTests.swift

### PR DESCRIPTION
**Pull request by marinkman**
## What?
- Removed spaces from all project folder/file names.

## Why?
- To close #42 

## Anything Else?
- Renamed the  PRODUCT_BUNDLE_IDENTIFIER  from "com.swiftcowboys.ios.sleeptracker.Sleep-Tracker" to "com.swiftcowboys.ios.sleeptracker.SleepTracker"
- Renamed the scheme from "Sleep Tracker" to "SleepTracker"

### CODE REVIEW

**Review by MikhailKosarev — APPROVED** [review]
You've done a great job! It's pleasing to the eye now :)
LGTM

File: Sleep Tracker/SleepTracker.xcodeproj/xcshareddata/xcschemes/SleepTracker.xcscheme [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dersim-davaod) [inline Sleep Tracker/SleepTracker.xcodeproj/xcshareddata/xcschemes/SleepTracker.xcscheme:1]: Is this a necessary change?
Is this file a result of renaming?

**Review by dersim-davaod — COMMENTED** [review]
Quick question:

File: Sleep Tracker/SleepTracker.xcodeproj/xcshareddata/xcschemes/SleepTracker.xcscheme (hunk 1 above)
Comment (marinkman) [reply Sleep Tracker/SleepTracker.xcodeproj/xcshareddata/xcschemes/SleepTracker.xcscheme:1]: This file appeared after the schema was renamed.

**Review by dersim-davaod — APPROVED** [review]
Superb! LGTM!

**Pull request merged**